core/variant/type_info.h

```diff
@@ -300,6 +300,7 @@ class BitField {
 	_FORCE_INLINE_ constexpr BitField(T p_value) { value = (int64_t)p_value; }
 	_FORCE_INLINE_ operator int64_t() const { return value; }
 	_FORCE_INLINE_ operator Variant() const { return value; }
+	_FORCE_INLINE_ BitField<T> operator*(const BitField<T> &p_b) const { return BitField<T>(value ^ p_b.value); }
```

Comment:
```suggestion
	_FORCE_INLINE_ BitField<T> operator^(const BitField<T> &p_b) const { return BitField<T>(value ^ p_b.value); }
```
I'd suggest the xor operator unless unavailable to not be confusing

Reply:
Might not have reached that point as the one use is in the platform code

Also I make these all the time